### PR TITLE
Add simplegmail

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
I opened a pull request up-stream to include the patch https://github.com/jeremyephron/simplegmail/pull/57

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/simplegmail/setup.patch [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+diff --git a/setup.py b/setup.py
+index a191fcf..9b97d44 100644
+--- a/setup.py
++++ b/setup.py
+@@ -12,7 +12,6 @@
+     packages=setuptools.find_packages(),
+     install_requires=[
+         'google-api-python-client>=1.7.3',
+-        'bs4>=0.0.1',
```
Comment (ocefpaf) [inline recipes/simplegmail/setup.patch:18]: BTW, we do have a "bs4" package that installs beautifulsoup4. Maybe we don't need this patch.

Comment (jan-janssen) [reply recipes/simplegmail/setup.patch:18]: I tried it before without the patch, but then the `pip check` complained.

Comment (ocefpaf) [reply recipes/simplegmail/setup.patch:18]: Ah, yeah. pip check is not too smart about that.

**Pull request merged**